### PR TITLE
8103 mission based operations

### PR DESCRIPTION
**Pull request by DevneyHamilton**

### CODE REVIEW

**Comment by DevneyHamilton** [comment]
Note: when login is stale, get ActionView::Template::Error at /en/confirm-login
No route matches {:action=>"index", :confirm=>true, :controller=>"operations", :locale=>:en, :mission_name=>nil, :mode=>nil}, possible unmatched constraints: [:mode] from admin view of operations. Where is the mode set for login confirmation?

**Comment by DevneyHamilton** [comment]
TODO: update manual testing instructions to reflect that operations are mission-based.

File: app/controllers/operations_controller.rb [hunk 1]
```diff
@@ -6,7 +6,11 @@ class OperationsController < ApplicationController
   load_and_authorize_resource
 
   def index
-    @operations = @operations.order(created_at: :desc)
+    @operations = if current_mission.present?
+                    @operations.for_mission(current_mission).order(created_at: :desc)
+                  else
+                    @operations.order(created_at: :desc) # Display ALL operations on server
```
Comment (smoyth) [inline app/controllers/operations_controller.rb]: Nice comment.

**Comment by smoyth** [comment]
Great job! No changes!

**Pull request merged**